product-page/src/App.js

```diff
@@ -0,0 +1,14 @@
+import './App.css'
+import ProductList from './features/products/ProductList'
+
+function App() {
```

Comment:
I would still change this to a function expression - I know this is how react default outputs this but just a suggestion.